examples/chunked_example_fast/frontend.lua

```diff
@@ -45,8 +45,10 @@ local function get_key_def(space_name, index_name)
 end
 
 local function decode_metainfo(buf)
-    -- Skip an array around a call return values.
     local len
+    -- Skip map header
+    msgpack:skip_request_header(buf)
```

Comment:
1. It is not about msgpack, the function domain is iproto / net.box.
2. It is about a response, not a request.
3. The function must not be no-op on tarantool-1.10, it is confusing.

Reply:
Agreed that `msgpack` is not perfect fit here, but where should we place this work-around? To the example itself (looks incorrect because work-around should be generic as it's 1.10 wide problem). I'm very open to suggestions?

Reply:
At least, it should be something under `tuple.merger` namespace. Raw ideas:

1. We can let the function be a part of the example. In fact, it is just simple wrapper around the `decode_map_header` function.
2. `tuple.merger.advance_to_iproto_data`.